molecule/default/Dockerfile_debian_9.j2

```diff
@@ -1,12 +0,0 @@
-# Molecule managed
```

Comment:
Is this no longer needed for this project or in general? This file is still present in the skeleton and the last time this file was touched was to revert a removal in https://github.com/cisagov/ansible-role-xfce-cool/commit/86d8db4c554a86b22daf0a1b65d4ff4f0b7df392.

Reply:
This file is needed anywhere our Ansible roles still support Debian 9 is still supported.  The default `molecule` foo uses Python 3, so we need to override that for Debian 9 since Debian 9 only supports CyHy.

Are you suggesting we drop support for Debian 9 in this Ansible role?  I was planning to maintain support for Debian 9 wherever possible until CyHy is no longer dependent on it, just in case, but i could be persuaded otherwise.

Reply:
You've removed the file so I was asking if that was desired. I was under the assumption that since it is testing Debian 9 we would still want this file.

Reply:
Ah, my wires were crossed.  I cut the Gordian knot in commit d248aa7.